client/src/components/TicketForm/TicketForm.jsx

```diff
@@ -84,8 +83,12 @@ class TicketForm extends Component {
 		}
 
 		if (this.state.description.value && this.state.location.value && this.state.contact.value) {
+			this.props.getCurrentUser();
```

Comment:
When the user clicks on the login button (in Nav.jsx), you can store the current user.